### ISSUE TITLE
Typo in client howto

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11158]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#1603|
|Type|defect|
|Created|2006-03-25 17:34:19Z|

_In many cases, the **protocl** only needs to connect to the server once_

<details><summary>Searchable metadata</summary>

```
trac-id__1603 1603
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1143308059000000 1143308059000000
changetime__1173360780000000 1173360780000000
version__ 
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="puzzlement's avatar" src="https://avatars.githubusercontent.com/u/94942?s=50" width="50" height="50">](https://github.com/puzzlement)|@puzzlement set owner to edsuom|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_2"></a>|@therve set status to `closed`|
|-|-|

(In [19782]) Correct typo in client howto

Author: exarkun
Reviewer: therve
Fixes [#11158](https://github.com/twisted/twisted/issues/11158)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
